.eslintrc.json

```diff
@@ -0,0 +1,21 @@
+{
+    "env": {
+        "commonjs": true,
+        "es2020": true,
+        "node": true
+    },
+    "extends": [
+        "airbnb", "prettier", "plugin:node/recommended"
+    ],
+    "parserOptions": {
+        "ecmaVersion": 11
+    },
+	"plugins": ["prettier"],
+    "rules": {
+		"prettier/prettier": ["error"],
+		"no-param-reassign": ["error", { "props": false }],
+		"no-underscore-dangle": "off",
+		"no-plusplus": "off",
+		"class-methods-use-this": "off"
+    }
+}
```

Comment:
maybe use 2 space indentation to keep things consistent

Reply:
Yeah, looks like tabs and spaces got mixed up. Will fix that